subsystems/mvdm/ntvdm/dos/dos32krnl/process.c

```diff
@@ -796,10 +796,40 @@ WORD DosCreateProcess(IN LPCSTR ProgramName,
         /* Those are handled by NTVDM */
         case SCS_WOW_BINARY:
         {
-            DisplayMessage(L"Trying to load '%S'.\n"
-                           L"WOW16 applications are not supported internally by NTVDM at the moment.\n"
-                           L"Press 'OK' to continue.",
-                           ProgramName);
+            CHAR AppName[50] = "\"%programfiles%\\otvdm\\otvdmw.exe\" ";
+            STARTUPINFOA si;
+            CHAR ExpName[256];
+            ExpandEnvironmentStringsA(AppName, ExpName, 255);
+            strcat(ExpName, "\"");         // Add double-quote before ProgramName
+            strcat(ExpName, ProgramName);  // Append Program name
+            strcat(ExpName, "\"");         // Add double-quote after ProgramName
+            PROCESS_INFORMATION pi;
+            ZeroMemory( &si, sizeof(si) );
+            si.cb = sizeof(si);
+            ZeroMemory( &pi, sizeof(pi) );
+
+            /* Create the process. */
+            if(CreateProcessA( NULL,   // No Application Name
```

Comment:
```suggestion
            ZeroMemory(&pi, sizeof(pi));
            ZeroMemory(&si, sizeof(si));
            si.cb = sizeof(si);

            /* Create the process. */
            if (CreateProcessA(NULL,   // No Application Name
```

Reply:
As long as the earth is circling around the sun, generations of developers will agree to disagree about how that kind of space-padding is done best :D

Reply:
Just follow the already-existing **consistent** code formatting in NTVDM and that's it!

Reply:
Fixed. Thanks.